binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="$(NuGetTargetMoniker.Contains('UAP')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
```

Comment:
because when you build a netstandard reference assembly, it should be compatible with uap as well. If we don't use this, then libraries which are currently picking the configuration of netstandard won't be in the uap targeting pack which is wrong.